tests/integration/google_cloud_storage/test_google_cloud_storage.py

```diff
@@ -222,8 +222,19 @@ def test_perform_google_cloud_storage_tests(
         listdir_parent = listdir_dataitem_parent.listdir()
         assert os.path.basename(self._object_path) not in listdir_parent
 
+    @pytest.mark.parametrize(
+        "setup_by, use_secrets",
+        [
+            ("credentials_file", False),
+            ("credentials_file", True),
+            ("serialized_json", False),
+            ("serialized_json", True),
+        ],
+    )
```

Comment:
Small style thing – independent parameters would normally be specified as:
```suggestion
    @pytest.mark.parametrize(
        "setup_by", ("credentials_file", "serialized_json"),
    )
    @pytest.mark.parametrize(
        "use_secrets", (False, True),
    )
```
to avoid repetition and communicate better the fact that all permutations are tried.

Reply:
nice idea!